docs/commands/ft.create.md

```diff
@@ -56,8 +56,14 @@ after the SCHEMA keyword, declares which fields to index:
 
  - `VECTOR` - Allows vector similarity queries against the value in this attribute. For more information, see [Vector Fields](/redisearch/reference/vectors).
 
- - `GEOSHAPE`- Allows polygon queries against the value in this attribute. The value of the attribute must follow [WKT notation](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry) a list of 2D points representing the polygon edges `POLYGON((x1 y1, x2 y2, ...)` separated by a comma. Current not support JSON multi-value and `SORTABLE` option.
-`
+ - `GEOSHAPE`- Allows polygon queries against the value in this attribute. The value of the attribute must follow a [WKT notation](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry) list of 2D points representing the polygon edges `POLYGON((x1 y1, x2 y2, ...)` separated by a comma. A `GEOSHAPE` field type can be followed by one of the following coordinate systems:
```

Comment:
The docs should specify the `COORD_SYSTEM` keyword, [I had to dig in the code](https://github.com/RediSearch/RediSearch/pull/3674/files#diff-4d570566161a8f0ca29c7e12a31bc46b08fa89960ff034b711235bdd897efd6fR152) to find that...